src/components/Pricing.tsx

```diff
@@ -136,6 +137,52 @@ function PriceSlider({
   );
 }
 
+function TeamPlanV2() {
+  const features: Feature[] = [
+    { title: 'Unlimited apps' },
+    { title: 'Unlimited developers' },
+    { title: '50K patch installs/month included' },
+    { title: 'Control exactly how much you spend' },
```

Comment:
```suggestion
    { title: 'Configurable spending limit' },
```

🤷

Reply:
@eseidel thoughts on this?

Reply:
I'm not sure we need to update the copy for teams at all?  AT least in terms of the benefits.  "PAYG" isn't really a feature so much as "table stakes" to those coming to us from never having heard of us.